### ISSUE TITLE
Angular Compilation Failed

### ISSUE THREAD

**Issue by PrashantSinghGour** [nhn/tui.editor#1974]
## Angular app not compiling.
I installed this in our project which is working on Angular 8, got the following error on compilation.

```
[1]
[1] ERROR in node_modules/@toast-ui/editor/types/editor.d.ts:269:45 - error TS1005: ',' expected.
[1]
[1] 269 export type SelectionPos = Sourcepos | [from: number, to: number];
[1]                                                 ~
[1] node_modules/@toast-ui/editor/types/editor.d.ts:269:57 - error TS1005: ',' expected.
[1]
[1] 269 export type SelectionPos = Sourcepos | [from: number, to: number];
[1]                                                             ~

[1] ** Angular Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
[1] ℹ ｢wdm｣: Failed to compile.
```

## To Reproduce
1. install angular 8
2. npm i @toast-ui/editor prosemirror-transform --save
3. ng serve (to compile)

## Additional context
When I changed 
`export type SelectionPos = Sourcepos | [from: number, to: number];` to
`export type SelectionPos = Sourcepos;`
it complied successfully.

**Comment by js87zz** [comment]
@PrashantSinghGour 
Our type definition is defined with typescript `v4.2.3`.
If your typescript version is lower than `v4.2.3`, the ts compiler cannot check the [labeled tuple](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements) because the labeled tuple is included above the `v4.0.0`.

If it is difficult to update the typescript version, it would be good to consider to remove library checks using [`skipLibCheck` option](https://www.typescriptlang.org/tsconfig#skipLibCheck).

**Comment by PrashantSinghGour** [comment]
@js87zz 
In our tsconfig we have already done that,
`   },
    "skipLibCheck": true,
  },
  "angularCompilerOptions": {
    "fullTemplateTypeCheck": true,
    "strictInjectionParameters": true
  }`
